src/Catalyst.jl

```diff
@@ -92,4 +92,10 @@ include("latexify_recipes.jl")
 include("graphs.jl")
 export Graph, savegraph, complexgraph
 
+# for creating compounds
+include("compound.jl")
+export @compound
+export components, iscompound, coefficients, component_coefficients, balance, get_stoich
```

Comment:
You are exporting non-existent functions here? (i.e. `balance` and `get_stoich`?) Also, let's not export `component_coefficients`.